test/frontend.js

```diff
@@ -259,24 +259,82 @@ describe('Frontend', function () {
     })
 
     it('Should create a group', async function () {
-      this.timeout(10000)
-      await n.click('#CreateGroup')
-      const created = await n
-        .insert('input[name="groupName"]', 'Test Group')
-        .insert('textarea[name="sharedValues"]', 'Testing this software')
-        .insert('input[name="groupName"]', 'Test Group')
-        .insert('input[name="incomeProvided"]', 200)
-        .select('select[name="contributionPrivacy"]', 'Very Private')
-        .click('button[type="submit"]')
-        .wait('#addButton')
-        .exists('#addButton')
-      should(created).equal(true)
+      const testName = 'Test Group'
+      const testValues = 'Testing this software'
+      const testIncome = 200
+      const testSetting = 60
+      this.timeout(15000) // It just takes some time
+      await n
+        .click('#CreateGroup')
+        // fill group data
+        .wait('input[name="groupName"]')
+        .insert('input[name="groupName"]', testName)
+        .click('#nextBtn')
+        .wait('textarea[name="sharedValues"]')
+        .insert('textarea[name="sharedValues"]', testValues)
+        .click('#nextBtn')
+        .wait('input[name="incomeProvided"]')
+        .insert('input[name="incomeProvided"]', testIncome)
+        .click('#nextBtn')
+        .wait('#rulesStep')
+        // set rules
+        .click('#changeRulesToggle')
+        .wait('input[name="changePercentage"]')
+        .insert('input[name="changePercentage"]')
+        .insert('input[name="changePercentage"]', testSetting)
+        .click('#approveToggle')
+        .wait('input[name="memberApprovalPercentage"]')
+        .insert('input[name="memberApprovalPercentage"]')
+        .insert('input[name="memberApprovalPercentage"]', testSetting + 10)
+        .click('#removeToggle')
+        .wait('input[name="memberRemovalPercentage"]')
+        .insert('input[name="memberRemovalPercentage"]')
+        .insert('input[name="memberRemovalPercentage"]', testSetting - 10)
+        .click('#nextBtn')
+        .wait('#privacyStep')
+        .click('#nextBtn')
+        // invite members
+        .wait('input[name="invitee"]')
+        .insert('input[name="invitee"]', username + '4')
+        .click('#addButton')
+        .wait('.invitee')
+        .wait(10000)
```

Comment:
No manual wait times allowed.

Reply:
🙈  I used it for debugging and forgot to remove. well, this explains the timeouts. removing.